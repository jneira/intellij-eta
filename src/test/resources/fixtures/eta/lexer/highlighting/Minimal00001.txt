ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Minimal00001')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITinline_prag ('{-#')
ITline_prag (' MINIMAL foo ')
ITline_prag ('#-}')
WHITE_SPACE ('\n    ')
ITvarid ('foo')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITvarid ('bar')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n\n')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo'')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvarid ('foo'')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITinline_prag ('{-#')
ITline_prag (' MINIMAL foo' ')
ITline_prag ('#-}')
WHITE_SPACE ('\n    ')
ITvarid ('bar'')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n\n')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo''')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvarid ('foo''')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITvarid ('bar''')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITinline_prag ('{-#')
ITline_prag (' MINIMAL foo'' ')
ITline_prag ('#-}')
WHITE_SPACE ('\n')
