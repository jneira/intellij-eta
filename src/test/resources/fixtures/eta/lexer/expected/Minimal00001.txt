ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Minimal00001')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvocurly ('')
ITminimal_prag ('{-# MINIMAL')
WHITE_SPACE (' ')
ITvarid ('foo')
WHITE_SPACE (' ')
ITclose_prag ('#-}')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarid ('foo')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarid ('bar')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n\n')
ITvccurly ('')
ITsemi ('')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo'')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvocurly ('')
ITvarid ('foo'')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITminimal_prag ('{-# MINIMAL')
WHITE_SPACE (' ')
ITvarid ('foo'')
WHITE_SPACE (' ')
ITclose_prag ('#-}')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarid ('bar'')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n\n')
ITvccurly ('')
ITsemi ('')
ITclass ('class')
WHITE_SPACE (' ')
ITconid ('HasFoo''')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n    ')
ITvocurly ('')
ITvarid ('foo''')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarid ('bar''')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITvarid ('a')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('String')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITminimal_prag ('{-# MINIMAL')
WHITE_SPACE (' ')
ITvarid ('foo''')
WHITE_SPACE (' ')
ITclose_prag ('#-}')
WHITE_SPACE ('\n')
ITvccurly ('')
ITsemi ('')